spec/system/viewing_an_application/history_spec.rb

```diff
@@ -121,4 +121,40 @@
       expect(first_row).to match('Evidence issue')
     end
   end
+
+  context 'with a completed application' do
+    before do
+      stub_request(
+        :put,
+        "#{ENV.fetch('DATASTORE_API_ROOT')}/api/v2/applications/#{crime_application_id}/complete"
+      ).to_return(body: LaaCrimeSchemas.fixture(1.0).read, status: 200)
+
+      user = User.create(
+        first_name: 'Fred',
+        last_name: 'Smitheg',
+        auth_oid: '976658f9-f3d5-49ec-b0a9-485ff8b308fa',
+        email: 'Fred.Smitheg@justice.gov.uk'
```

Comment:
Wondering if we should be using @example.com - as this could be a real email address

Reply:
I don't think it will be as the 'eg' is suffixed to me "example" or "e.g."

Reply:
I have just heard that it's best practice not to use real domains in emails in tests

Reply:
I think this iss good point - lets change this.